test/bddtests/filehandler_steps.go

```diff
@@ -171,12 +220,45 @@ func (d *FileHandlerSteps) saveDocIDToVariable(varName string) error {
 	return nil
 }
 
+func (d *FileHandlerSteps) setJSONPatchVar(varName, patch string) error {
+	var p []interface{}
+	err := json.Unmarshal([]byte(patch), &p)
+	if err != nil {
+		panic(err)
+	}
+
+	obj, err := bddtests.ResolveVars(p)
+	if err != nil {
+		panic(err)
+	}
+
+	bytes, err := json.Marshal(obj)
+	if err != nil {
+		panic(err)
+	}
+
+	logger.Infof("Setting variable [%s] to JSON patch %s", varName, bytes)
+
+	bddtests.SetVar(varName, string(bytes))
+
+	return nil
+}
+
 func (d *FileHandlerSteps) getOpaqueDocument(content string) string {
 	doc, _ := document.FromBytes([]byte(content))
 	bytes, _ := doc.Bytes()
 	return string(bytes)
 }
 
+func (d *FileHandlerSteps) getUpdateRequest(uniqueSuffix string, patch jsonpatch.Patch) ([]byte, error) {
+	return helper.NewUpdateRequest(&helper.UpdateRequestInfo{
+		DidUniqueSuffix: uniqueSuffix,
+		Patch:           patch,
+		UpdateOTP:       docutil.EncodeToString([]byte(updateOTP)),
```

Comment:
Just bear in mind that if you are doing multiple updates you will also have to provide NextUpdateOTP.